### DISCUSSION TITLE
Did my plugins disappear for a reason?

### DISCUSSION THREAD

**Discussion by BlueNalgene** [ampache/ampache#3994] [Q&A] [answered]
I was chasing down why I've never seen the shoutbox on my install, and I noticed that most of my plugins disappeared when I updated from 6.2 to 6.6.  The only thing left in the server options is Last.FM.  Do I need to select something in the config now to enable them?  The release notes only indicate that Flattr was removed recently.

My install was a git pull from release branch 6.

![image](https://github.com/user-attachments/assets/ba2d64e9-f990-4f40-b839-4626e296e8ac)

**Comment by lachlan-00** [comment, marked as answer]
I generally won't change plugin state and especially won't delete the preferences on purpose.

Are they activated on the manage plugins page?

They should only get deleted when the plugin is uninstalled, I had a quick look but I can't see an obvious situation where they're going to be deleted.

user_preferences will get garbage collected when the preference is missing from the main preference table so that's a possible reason.

**Comment by BlueNalgene** [comment]
> Are they activated on the manage plugins page?

This particular PEBKAC issue has been identified.  Somehow I collapsed my Modules section and I never saw this.

For anyone else who is like me, you need to activate each plugin you want on a new install.  You can find this by logging in as admin, clicking the Admin tab on the lefthand menu (the icon looks kinda like a refrigerator).  Under "Modules", click "Manage Plugins".
